pkg/vault/contracts/test/VaultMock.sol

```diff
@@ -236,19 +235,18 @@ contract VaultMock is IVaultMainMock, Vault {
         }
     }
 
-    function manualSetPoolTokenBalances(
+    function manualSetPoolTokensAndBalances(
         address pool,
         IERC20[] memory tokens,
         uint256[] memory tokenBalanceRaw,
         uint256[] memory tokenBalanceLiveScaled18
     ) public {
-        EnumerableMap.IERC20ToBytes32Map storage poolTokenBalances = _poolTokenBalances[pool];
+        mapping(uint256 => bytes32) storage poolTokenBalances = _poolTokenBalances[pool];
```

Comment:
I know this comes from before, but maybe we should consider sanitizing the inputs to make sure at least that the arrays have the same length, even if it's a mock.